test/framework/alertmanager.go

```diff
@@ -336,8 +329,8 @@ func (f *Framework) SendAlertToAlertmanager(ns, n string, start time.Time) error
 		return err
 	}
 
-	var postAlertResp amAPIPostAlertResp
-	request := ProxyPostPod(f.KubeClient, ns, n, "api/v1/alerts", string(b))
+	var postAlertResp models.PostableAlerts
```

Comment:
We don't need (anymore?) to unmarshal the response into `postAlertResp` because the server replies with an empty body and `200 OK` in case of success.